docs/image-sizes.md

```diff
@@ -0,0 +1,40 @@
+# Images Sizes
+
+You can define additional image thumbnail sizes with `add_image_size`.
+
+The custom thumbnail sizes should be declared in the callback function to the `after_setup_theme` action.
+
+Example:
+```php
+add_action( 'after_setup_theme', __NAMESPACE__ . '\\theme_setup' );
+function theme_setup() {
+    add_image_size( 'category-thumb', 300, 9999 ); // 300 pixels wide (and unlimited height)
```

Comment:
We need more examples here of all the different types of sizes you can generate, so that includes cropped images. This is a bit too light touch as it is I think, I shouldn't have to visit the WP links below to understand what this does and how to use it.